src/asm.rs

```diff
@@ -107,26 +107,12 @@ enum ConstraintOrRegister {
 
 
 impl<'a, 'gcc, 'tcx> AsmBuilderMethods<'tcx> for Builder<'a, 'gcc, 'tcx> {
-    fn codegen_llvm_inline_asm(&mut self, ia: &LlvmInlineAsmInner, outputs: Vec<PlaceRef<'tcx, RValue<'gcc>>>, inputs: Vec<RValue<'gcc>>, span: Span) -> bool {
-        if ia.asm.as_str().is_empty() && outputs.is_empty() {
-            // TODO(@Commeownist): there's one use of `llvm_asm` in rustc sysroot we can't get rid of just yet.
-            // Fortunately, it's used as a simple black box to make sure that inputs are not optimized away.
-            // Let's just emulate it.
-            let block = self.llbb();
-            let extended_asm = block.add_extended_asm(None, "");
-            for input in inputs {
-                extended_asm.add_input_operand(None, "r", input);
-            }
-            extended_asm.add_clobber("memory");
-            extended_asm.set_volatile_flag(true);
-        } 
-        else {
-            // TODO(@Commeownist): switch to `struct_span_err_with_code` 
-            // once we get this merged into rustc
-            self.sess().struct_span_err(span, "GCC backend does not support `llvm_asm!`")
-                .help("consider using the `asm!` macro instead")
-                .emit();
-        }
+    fn codegen_llvm_inline_asm(&mut self, _ia: &LlvmInlineAsmInner, _outputs: Vec<PlaceRef<'tcx, RValue<'gcc>>>, _inputs: Vec<RValue<'gcc>>, span: Span) -> bool {
+        // TODO(@Commeownist): switch to `struct_span_err_with_code` 
+        // once we get this merged into rustc
```

Comment:
I don't think that is worth it given that `llvm_asm!` will be removed in the future.

Reply:
Do you want me to remove the comment? I don't understand, could you please explain?

Reply:
> Do you want me to remove the comment?

Yes